kcidb/monitor/spool/__init__.py

```diff
@@ -99,9 +99,7 @@ def post(self, notification, timestamp=None, due=None):
                             datetime.datetime.now(datetime.timezone.utc).
             due:            An "aware" datetime.datetime object specifying the
                             time notification should be sent out,
-                            or None to use
-                            datetime.datetime.now(datetime.timezone.utc).
-
+                            or None to use the timestamp.
```

Comment:
Could you change `the timestamp` to `"timestamp"` so it's clearer we're referring to the specific argument?